workspace.json

```diff
@@ -8,7 +8,7 @@
     "eslint-plugin": "packages/eslint-plugin",
     "nx-plugin": "packages/nx-plugin",
     "orchestrator-contracts": "contracts/orchestrator-contracts",
-    "serverless-cdk": "packages/serverless-cdk",
+    "serverless-cdk-plugin": "packages/serverless-cdk-plugin",
```

Comment:
We can make the renaming in the first commit, another commit is not necessary

Reply:
done